neurolib/models/fhn/loadDefaultParams.py

```diff
@@ -78,6 +78,10 @@ def loadDefaultParams(Cmat=None, Dmat=None, seed=None):
     params.x_ext = np.ones((params.N,))
     params.y_ext = np.zeros((params.N,))
 
+    # time-dependent stimulation inputs
+    params.x_stim = 0.
+    params.y_stim = 0.
```

Comment:
Instead of adding a new parameter, the old ones should be extended.

Reply:
if we keep one set of external stimulation variables only, it is not entirely analogous to the ALN model, where we have "mue_ext_mean", "mui_ext_mean" as constant inputs that enable to determine the location in state-space, and "ext_exc_current", "ext_inh_current", which are time-dependent inputs. Also, there wouldn't be parameters to define the state-space location (which at least for the WC model might be problematic).